Runtime/Scripts/DrawHTMLOverCanvas.cs

```diff
@@ -62,8 +54,10 @@ private void AlignHTMLOverlay()
 		{
 
 			Canvas canvas = GetComponent<RectTransform>().root.GetComponent<Canvas>();
-			float canvasheight = canvas.GetComponentInParent<RectTransform>().rect.height;
-			float canvaswidth = canvas.GetComponentInParent<RectTransform>().rect.width;
+			var canvasScaleFactor = canvas.scaleFactor;	
+			float canvasheight = canvas.GetComponentInParent<RectTransform>().rect.height * canvasScaleFactor;
```

Comment:
Minuscule optimalisatie: GetComponentInParent resultaat cachen. En is dit niet hetzelfde als De GetComponent<RectTransform> in regel 56?

Reply:
gecached. die ander is recttransform van t canvas; maar heb die nu ook gecached 👍